cmd/tnf/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"path"
+	"text/template"
+
+	"github.com/spf13/cobra"
+)
+
+type myHandler struct {
+	Handlername string
+}
+
+var (
+	pathrelativetoroot string
+	handlerDirectory   string
+	handlername        string
+
+	rootCmd = &cobra.Command{
+		Use:   "tnf",
+		Short: "A CLI for creating, validating, and test-network-function tests.",
+	}
+
+	generate = &cobra.Command{
+		Use:   "generate",
+		Short: "generator tool for various tnf artifacts.",
+	}
+
+	handler = &cobra.Command{
+		Use:   "handler",
+		Short: "adding new handler.",
+		RunE:  generateHandlerFiles,
+	}
+)
+
+func generateHandlerFiles(cmd *cobra.Command, args []string) error {
+	handlername = args[0]
+	pathrelativetoroot = path.Join("..", "..")
```

Comment:
this should be : 
**pathrelativetoroot = path.Join("", "")**
are you sure this code work ? I could not make it work on my machine